pkg/gce-pd-csi-driver/controller.go

```diff
@@ -1186,9 +1234,17 @@ func (gceCS *GCEControllerServer) ControllerExpandVolume(ctx context.Context, re
 		}
 		return nil, LoggedError("ControllerExpandVolume error repairing underspecified volume key: ", err)
 	}
-
+	sourceDisk, err := gceCS.CloudProvider.GetDisk(ctx, project, volKey, gce.GCEAPIVersionV1)
```

Comment:
What happens if the GetDisk call failed?

Reply:
Added error handling for GetDisk calls.